src/subscription_manager/repofile.py

```diff
@@ -610,6 +610,10 @@ def fix_content(self, content):
         if zypper_cont['gpgkey'] in ['https://', 'http://']:
             del zypper_cont['gpgkey']
 
+        # make sure gpg key download doesn't fail because of private certs
+        if zypper_cont['gpgkey'] and sslverify and sslverify in ['1']:
```

Comment:
I know that you can see `sslverify and sslverify in ['1']` on line 642, but why not just use `sslverify == '1'`?

Reply:
I have re-used the code in https://github.com/candlepin/subscription-manager/blob/f29bbc8e68117fb7a530426d4dedd46cd5556fd6/src/subscription_manager/repofile.py#L633 - just to be sure that I don't break anything.